tests/assert.go

```diff
@@ -0,0 +1,47 @@
+//
```

Comment:
Do we need this? 
see "github.com/stretchr/testify/require"
"github.com/stretchr/testify/assert"
https://github.com/stretchr/testify

Reply:
Ok, I will use that.